x/ccv/provider/migrations/migrator.go

```diff
@@ -0,0 +1,32 @@
+package migrations
+
+import (
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+
+	providerkeeper "github.com/cosmos/interchain-security/v3/x/ccv/provider/keeper"
+	v3 "github.com/cosmos/interchain-security/v3/x/ccv/provider/migrations/v3"
+)
+
+// Migrator is a struct for handling in-place store migrations.
+type Migrator struct {
+	providerKeeper providerkeeper.Keeper
+	paramSpace     paramtypes.Subspace
+}
+
+// NewMigrator returns a new Migrator.
+func NewMigrator(providerKeeper providerkeeper.Keeper, paramSpace paramtypes.Subspace) Migrator {
+	return Migrator{providerKeeper: providerKeeper, paramSpace: paramSpace}
+}
+
+// Migrating consensus version 1 to 2 is a no-op.
+// <igrating from v1 -> v2 -> v3  will require multiple state breaking changes and migrations.
+// First run provider@v2.x.y in production to migrate from v1 -> v2. Then use provider@v3.x.y to migrate from v2 -> v3.
```

Comment:
```suggestion
// First run provider@v2.x.y in production to migrate from consensus version 1  to 2. 
// Then, in order to migrate to consensus version 3,  first upgrade to provider@v3.x.y.
```

Reply:
Is it possible to upgrade to consensus version 3 directly from provider@v2.x.y?

Reply:
Yes. `Migrate2to3` does that